components/navbar/navbar.css

```diff
@@ -0,0 +1,8 @@
+.navigation-container > nav {
```

Comment:
Just suggestion in class naming it could be .nav-container as below you declared .nav-logo